vite.config.js

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+})
```

Comment:
PR for Meghana

Code functionality is working as expected.

Correctly adhered to coding styling guidelines and bootstrap react.
Overall code structure is well-defined and easily maintainable and readable.